### ISSUE TITLE
cpc does not seem to be able to handle new object, XcmsExperiment, from xcms.

### ISSUE THREAD

**Issue by dongdongdong7** [krispir/cpc#5]
I'm trying to combine cpc with xcms 4.0 processes but I realized that both use different objects. And I didn't find any function that converts XcmsExperiment to XCMSnExp.

**Comment by jorainer** [comment]
that's correct. unfortunately we did not yet have the time to implement the method for `XcmsExperiment` - and there is also no function yet to coerce from `XCMSnExp` to `XcmsExperiment`.

**Comment by dongdongdong7** [comment]
@jorainer Thank you for your answer.The new framework of XCMS works very well and clearly, but unfortunately many of the previous tools cannot be integrated directly, because of the new objects. I think an object conversion function is very necessary!
